src/redux/epics/event.epics.js

```diff
@@ -5,14 +5,19 @@ import 'rxjs/add/operator/mergeMap';
 
 const BASE_ENDPOINT = `https://www.eventbriteapi.com/v3/events/search/?token=${EVENTBRITE_API_KEY}&location.address=toronto&categories=`;
 
-export const getEventEpic = (action$, store) =>
+export const returnEventActionOnLocationChange = (action$) =>
+  action$.ofType('@@router/LOCATION_CHANGE')
+    .filter(action => action.payload.pathname.includes('category/'))
```

Comment:
I think you might be reviewing an old version of the code. I have already updated this to:
```
const reCategoryPath = /^.*\/category\/\d\d\d$/;

export const returnEventActionOnLocationChange = (action$) =>
  action$.ofType('@@router/LOCATION_CHANGE')
    .filter(action => reCategoryPath.test(action.payload.pathname))
```